opm/input/eclipse/Python/PyRunModule.cpp

```diff
@@ -21,13 +21,16 @@ error BUG: The PyRunModule.hpp header should *not* be included in a configuratio
 #endif
 
 
+#include <opm/common/OpmLog/OpmLog.hpp>
 #include <opm/input/eclipse/EclipseState/EclipseState.hpp>
 #include <opm/input/eclipse/Schedule/Schedule.hpp>
 #include <opm/input/eclipse/Schedule/SummaryState.hpp>
 
 #include "PyRunModule.hpp"
 
-#include <filesystem>
+#include <iostream>
```

Comment:
This should not be needed.

(I thought I made that comment yesterday. Apparently, not. Maybe forget to press the right burron.)